instaloader/instaloader.py

```diff
@@ -681,7 +679,8 @@ def download_highlights(self,
             name = user_highlight.owner_username
             highlight_target = (filename_target
                                 if filename_target
-                                else Path(name) / Path(user_highlight.title))  # type: Union[str, Path]
+                                else Path(_PostPathFormatter.sanitize_path(name))
+                                / _PostPathFormatter.sanitize_path(user_highlight.title))  # type: Union[str, Path]
```

Comment:
```suggestion
                                else (Path(_PostPathFormatter.sanitize_path(name)) /
                                      _PostPathFormatter.sanitize_path(user_highlight.title)))  # type: Union[str, Path]
```

Reply:
Thanks! Yeah I'm actually not sure which one we prefer (can't find any other link break at binary operator), so I followed https://www.flake8rules.com/rules/W504.html

Do you think we should use after-operator instead? I can change it.

Reply:
Indeed, having the operator at the beginning of the line might be better. Whatever you prefer :)

Reply:
> Indeed, having the operator at the beginning of the line might be better. Whatever you prefer :)

I will just leave it as what you suggested I think. Thanks!